Cookbook/Style-guide/README.md

```diff
@@ -381,10 +381,16 @@ Use self only when required by the compiler (in `@escaping` closures, or in init
 
 ## Computed Properties
 
-For conciseness, if a computed property is read-only, omit the get clause. The get clause is required only when a set clause is provided.
+For conciseness, if a computed property is read-only, omit the get clause. The get clause is required only when a set clause is provided. For single-expressions where the context is clear, both explicit and implicit returns can be used:
 
 **Preferred**:
 ```swift
+var diameter: Double {
```

Comment:
how about `var diameter: Double { radius * 2 }`? should we allow this or discourage?

Reply:
It looks readable and easily understandable to me. I'd vote 👍

Reply:
+1 for adding one-liner

Reply:
Added in f91870b